packages/react-magma-dom/src/components/Table/Table.tsx

```diff
@@ -45,6 +46,11 @@ export interface TableProps extends React.HTMLAttributes<HTMLTableElement> {
    */
   minWidth?: number;
   rowCount?: number;
+  /**
+   * Title that appears above the Datagrid
```

Comment:
I wonder if we should be clear that this will be inserted inside `<caption>`. Something like 

- The title or caption of a table inside a `<caption>` HTML element that provides the table an accessible description

(this applies to both Table and DataGrid)